lib/util/isFirstLetterCapitalized.js

```diff
@@ -9,8 +9,7 @@ function isFirstLetterCapitalized(word) {
   if (!word) {
     return false;
   }
-  const firstLetter = word.charAt(0);
-  return firstLetter.toUpperCase() === firstLetter;
+  return /^[A-Z]/.test(word);
```

Comment:
a-z aren't the only letters that can be capitalized; i don't think this is a correct change.

Reply:
Good catch - I was too dependent on the existing unit tests for this function.
It'd be great if we could avoid doing some string manipulations here, so it could be that we can use the regex as a fast path, but preserve the chatAt & toUpperCase as a slow path.